Source/missiles.cpp

```diff
@@ -1907,7 +1907,7 @@ void AddTeleport(Missile &missile, Point dst, Direction /*midir*/)
 			Point target = dst + Displacement { CrawlTable[ck - 1], CrawlTable[ck] };
 			if (!InDungeonBounds(target) || IsTileSolid(target))

```

Comment:
Aren't these conditions also covered by `PosOkPlayer`?

Reply:
no clue, the function that had it was in anonymous namespace so I copy/pasted it